### DISCUSSION TITLE
Board 6.1.0 Discussion

### DISCUSSION THREAD

**Discussion by kagouraki** [peterantypas/maiana#128] [Q&A]
Hello Peter and other Folks, here is my story!

I know you dont support old boards but when the 6.1.0 version was released i bought my self pcb's and components for 2 boards because i wanted to test my skills and try to build it. When i finally had all the parts i had no time so the whole box was left in a drawer until two weeks ago. 
I assembled the two boards and after the oven they had some bridges which i fixed with the iron and flux, I checked under the microscope everything and everything seems to be connected and no bridges or whatsoever. 
Ι compiled the firmware in eclipse from the board7 branch and in the bsp.hpp file i edited the line 
```
#ifndef BOARD_REV
#define BOARD_REV 61
#endif
```
to match my 6.1.0 board.
I uploaded the .bin file from the SWD with st-link v2. 
I measured all the pads for correct voltages (vin, 3.3, 1.8) but i dont know what to measure in the V_PA pad.
After powering the board, it sends all the nmea to the serial so i think the firmware is the correct one. 
I programmed my ais data and after the reboot they are shown in the beginning so they are stored.
So the problem is when i go to my balcony and wait for a fix and ais targets, nothing happens. I have also a 10.5.0 board which i bought from you and works flawlessly almost a year now. From the 10.5.0 i see ais targets and i can obtain a fix. From the 6.1.0 i cant see any ais targets and after 5 minutes it took a fix but only one or two satelites and the fix had no good quality(was bumping around).
After that, i connected the 6.1.0 to my LDTZ spectrum analyzer to see with the "tx test" command if the unit is responsive, but when i connect the spectrum analyzer with 30db attenuation i get only noise from -80db to 0 db. That's strange...

At last i want to mention that both 6.1.0 boards i build behave the exact same way. So i have hope that maybe they will work ha ha ha

So my questions are :

1) what do i measure in the V_PA pad to see if everything works?

2)Can you compile and send me a .bin file for board 6.1.0 which is for sure working?(maybe i have a firmware issue)

3)Are there any other steps or things to check to see if i can make this board work?

If you or someone want to share knowledge this discussion will be awesome ! 

George
![329156220_590250262970285_515687641809212542_n](https://user-images.githubusercontent.com/6173967/224106725-03a39594-1821-4671-895e-b690d8c2fee6.jpg)

**Comment by peterantypas** [comment]
I'm afraid I'm not in a position to even build firmware for such an old board. For starters, I don't have any of them to test on. So you may have to do some debugging.

V_PA should be somewhere just above 8V.

If the system booted but there's no reception, it could be a number of issues, starting with the RF switch and its control lines to the data and clock lines from the RF ICs to the MCU. Also, this board has no external LNA compared to the 10.x series. Finally, one of the reasons why I abandoned this design is because the last revision of this Telit module has a DC/DC switch which literally craps all over the VHF band, and it's millimeters away from the RX front end. Also, I had many problems soldering the Telit modules.

**Reply by kagouraki** [reply]
That's perfect ! So i will put them to my "i can assemble boards until this level" drawer for my ancestors to find some day 😂. Thank you very much for your answer Peter and for this awesome project !
